### PR TITLE
Add Nunchuk (Desktop) connection details

### PR DESCRIPTION
**Pull request by bakaoh**
Adds connection instructions for Nunchuk Wallet Desktop (using Electrum)

### CODE REVIEW

**Review by lukechilds — APPROVED** [review]

File: src/components/ConnectWallet/Wallets/Nunchuk.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <connection-details name="Nunchuk Wallet" requires="electrum">
+    <step-list>
+      <step>
+        Open Nunchuk Wallet on your phone or your desktop.
+      </step>
+      <step>
+        Go to <span class="font-weight-bold">"Settings"</span>.
```
Comment (lukechilds) [inline src/components/ConnectWallet/Wallets/Nunchuk.vue:8]: On mobile this is called "Account" not "Settings".

File: src/components/ConnectWallet/Wallets/Nunchuk.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      </step>
+      <step>
+        Select <span class="font-weight-bold">"Mainnet Server"</span> and enter
+        <input-copy
+          :value="`${urls.bitcoin.electrum.address}:${urls.bitcoin.electrum.port}`"
+          auto-width
+        ></input-copy>
+      </step>
+      <step>
+        Disable <span class="font-weight-bold">"Set SSL Certificate"</span>.
```
Comment (lukechilds) [inline src/components/ConnectWallet/Wallets/Nunchuk.vue:22]: This doesn't appear for me on mobile.

File: src/components/ConnectWallet/Wallets/Nunchuk.vue [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      </step>
+      <step>
+        Disable <span class="font-weight-bold">"Enable TOR Proxy"</span>.
+      </step>
+      <step>
+        Close <span class="font-weight-bold">"Settings"</span>
```
Comment (lukechilds) [inline src/components/ConnectWallet/Wallets/Nunchuk.vue:28]: On mobile you need to click "Save netowrk settings" not close.

**Review by lukechilds — CHANGES_REQUESTED** [review]
It looks like the instructions are different for mobile and desktop. Maybe we should list them seperately?

Also, is Tor supposed to work on mobile? It doesn't appear to work for me.

**Comment by hugohn** [comment]
@lukechilds good catch! We're actually in the middle of a desktop redesign, which includes updating the Settings screen. It will be similar to what you currently see on mobile. The desktop build with the Settings UI changes will be out in a few days. So maybe it's okay with 1 component? WDYT? 

@bakaoh let's update the copy to reflect the latest design.

As for Tor, we have not yet hooked it up to the mobile UI. It's one of our TODOs.

**Comment by lukechilds** [comment]
Ok got it.

We're showing the Tor electrum address here, so bear in mind the instructions are gonna fail for mobile users currently if Tor isn't working for them. If you wanna ship it like this anyway and implement it later, that's also fine with me.

**Pull request merged**